docs/chapters/introduction/content.uk-UA.md

```diff
@@ -1,10 +1,5 @@
 # Короткий вступ
 
-Давайте розпочнемо з добрих новин: криві Безьє, про які ми говоритимемо, ви зможете побачити далі на графіках. Ці криві розпочинаються у якійсь певній точці, і закінчуються у якійсь певній точці.  Їх кривизна залежить від однієї або кількох "проміжних" контрольних точок. Зараз, оскільки всі графіки на цій сторінці інтерактивні, поекспериментуйте трохи з цими кривими. Клікніть на точку мишкою й потягніть - так ви зможете відчути, як форма кривої змінюється в залежності від ваших дій.  
-
-<div className="figure">
-  <Graphic inline={true} title="Квадратичні криві Безьє" setup={ this.drawQuadratic } draw={ this.drawCurve }/>
-  <Graphic inline={true} title="Кубічні криві Безьє" setup={ this.drawCubic } draw={ this.drawCurve }/>
-</div>
+Давайте розпочнемо з добрих новин: криві Безьє, про які ми говоритимемо, ви зможете побачити далі на графіках. Ці криві розпочинаються у якійсь певній точці, і закінчуються у якійсь певній точці. Їх кривизна залежить від однієї або кількох "проміжних" контрольних точок. Зараз, оскільки всі графіки на цій сторінці інтерактивні, поекспериментуйте трохи з цими кривими. Клікніть на точку мишкою й потягніть - так ви зможете відчути, як форма кривої змінюється в залежності від ваших дій.  
```

Comment:
it looks like the example code has gone missing now, though =)

admittedly that's in part github's fault, it's showing previews instead of source code for https://github.com/Pomax/BezierInfo-2/blob/master/docs/chapters/introduction/content.en-GB.md so looking at the raw content, or opening the file locally, will show you the "real" content: https://raw.githubusercontent.com/Pomax/BezierInfo-2/master/docs/chapters/introduction/content.en-GB.md

The only thing that's gone missing looks to be the graphics code though:

```suggestion
Давайте розпочнемо з добрих новин: криві Безьє, про які ми говоритимемо, ви зможете побачити далі на графіках. Ці криві розпочинаються у якійсь певній точці, і закінчуються у якійсь певній точці. Їх кривизна залежить від однієї або кількох "проміжних" контрольних точок. Зараз, оскільки всі графіки на цій сторінці інтерактивні, поекспериментуйте трохи з цими кривими. Клікніть на точку мишкою й потягніть - так ви зможете відчути, як форма кривої змінюється в залежності від ваших дій.  

<div class="figure">
  <graphics-element title="A quadratic Bézier curve" src="./quadratic.js"></graphics-element>
  <graphics-element title="A cubic Bézier curve" src="./cubic.js"></graphics-element>
</div>
```

of course, the titles will need retranslating but thankfully they're short =D

Reply:
Yep, I was confused by Github :)